lib/containers/urls.pm

```diff
@@ -293,6 +293,16 @@ sub get_suse_container_urls {
     return (\@untested_images, \@released_images);
 }
 
+# Get single image from CONTAINER_IMAGE_TO_TEST or untested image from get_suse_container_urls()
+sub get_container_image_to_test {
+    my $images_to_test;
+    unless ($images_to_test = get_container_url_from_var('CONTAINER_IMAGE_TO_TEST')) {
```

Comment:
```suggestion
    unless (my $images_to_test = get_container_url_from_var('CONTAINER_IMAGE_TO_TEST')) {
```

Reply:
No, this is not possible as we work with `$images_to_test` either way. I tested your suggestion tho [here](http://pdostal-server.suse.cz/tests/13776) tho.

Reply:
shit man. i missed that reused later. ignore